### ISSUE TITLE
Support wait in ec2_elb_lb

### ISSUE THREAD

**Issue by sheldonh** [ansible/ansible#11598]
##### Issue Type:

Feature Idea
##### Ansible Version:

```
ansible 2.0.0 (devel 0b035a4e35) last updated 2015/07/15 10:39:42 (GMT +200)
  lib/ansible/modules/core: (detached HEAD 291fef3b34) last updated 2015/07/15 10:39:52 (GMT +200)
  lib/ansible/modules/extras: (detached HEAD ff2386faf4) last updated 2015/07/15 10:39:52 (GMT +200)
  v1/ansible/modules/core: (detached HEAD f8d8af17cd) last updated 2015/07/08 13:25:41 (GMT +200)
  v1/ansible/modules/extras: (detached HEAD 495ad450e5) last updated 2015/07/08 13:25:47 (GMT +200)
  configured module search path = None
```
##### Ansible Configuration:

None
##### Environment:

N/A
##### Summary:

Please support `wait: True` on the `ec2_elb_lb` module, so that, in the `state: absent` case, it waits for the ELB to have been successfully removed before completing.
##### Steps To Reproduce:

Given an ELB called `MyELB` in a security group called `MySecurityGroup`, try to apply the following `playbook.yml`:

``` yaml
-
  hosts: localhost
  connection: local
  gather_facts: False
  tasks:
    -
      name: Destroy the ELB
      ec2_elb_lb:
        name: "MyELB"
        state: absent
    -
      name: Destroy the security group
      ec2_group:
        description: "My security group"
        name: "MySecurityGroup"
        state: absent
```

``` shell
ansible-playbook -i /dev/null playbook.yml
```
##### Expected Results:
- The playbook should complete successfully the first time it is run.
- The ELB and security group should not exist once the playbook completes.
##### Actual Results:
- The playbook fails in the `ec2_group` action the first time it is run.
- The ELB and security group exist once the playbook completes.
- After some time, rerunning the playbook produces the expected results.

The `ec2_group` failure is:

```
fatal: [localhost]: FAILED! => {
    "changed": false, 
    "failed": true, 
    "invocation": {
        "module_args": {
            "description": "My security group", 
            "name": "MySecurityGroup, 
            "state": "absent"
        }, 
        "module_name": "ec2_group"
    }, 
    "msg": "Unable to delete security group 'SecurityGroup:MySecurityGroup' - EC2ResponseError: 400 Bad Request\n<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<Response><Errors><Error><Code>DependencyViolation</Code><Message>resource sg-37222a52 has a dependent object</Message></Error></Errors><RequestID>32e811dd-2067-4373-ae4b-33a166bc0b12</RequestID></Response>"
}
```

**Comment by sheldonh** [comment]
Actually, it's not as simple as I thought. The reason the `ec2_group` task fails is that there is still an eni associated with the security group. The eni is created implicitly by AWS when the ELB is created, and it may be too much to expect Ansible to consider that.

**Comment by ryansb** [comment]
This feature was implemented in Ansible 2.1 and will not be backported to older versions.

**Comment by sheldonh** [comment]
Ah, nice. Thanks.
